build/config.props

```diff
@@ -19,7 +19,7 @@
 
     <!-- ** Change for each new preview/rc -->
     <!-- Check the VS schedule and manually enter a preview number here that makes sense. -->
-    <ReleaseLabel Condition=" '$(ReleaseLabel)' == '' ">preview.1</ReleaseLabel>
+    <ReleaseLabel Condition=" '$(ReleaseLabel)' == '' ">preview.3</ReleaseLabel>
```

Comment:
1. https://github.com/NuGet/NuGet.Client/pull/6002
2. we never use preview.3 because it's typically the last preview we insert into VS before it ships GA.  How do we make this part of our team institutional knowledge, so we stop making this mistake?

Reply:
> How do we make this part of our team institutional knowledge, so we stop making this mistake?

Thanks, Andy, for the feedback. It looks like you have added a comment `Preview 3 is typically the last "main branch" preview, so we start using rc at this time` to https://github.com/NuGet/NuGet.Client/pull/6002 PR. I am good with this comment. 

https://github.com/NuGet/NuGet.Client/pull/6002/commits/0dccb4f30ade28251bdb22cd9b774a424e3bae26#diff-bd4ef9ba7459727e37706ab7197bdfaa3868ec01297f168ff0b6e243f087d239R22

I am going to close this PR.